conda_build/conda_interface.py

```diff
@@ -165,7 +165,8 @@ def default(self, obj):
     cc_conda_build = context.conda_build if hasattr(context, 'conda_build') else {}
 
     # disallow softlinks.  This avoids a lot of dumb issues, at the potential cost of disk space.
-    os.environ['CONDA_ALLOW_SOFTLINKS'] = 'false'
+    # HINT: Keep the str() for py2 running on windows, see https://github.com/conda/conda-build/issues/1865
+    os.environ[str('CONDA_ALLOW_SOFTLINKS')] = str('false')
```

Comment:
Agreed, just thinking that when py2 compat is removed, it is much better to look for `PY2` conditionals, so the code is more organized.